.github/workflows/lint.yml

```diff
@@ -19,9 +19,6 @@ jobs:
       - uses: actions/checkout@v4
         with:
           show-progress: false
-      - uses: actions/setup-python@v5
```

Comment:
@hoodmane I know this was added intentionally, but the Ubuntu 24.04 image already comes with python 3.12. I'm open to keeping this if you see a need for it despite that, or if the lint script would benefit from python 3.13 (developers won't have access to 3.13 for now so I guess that's a reason to not depend on it for now).

Reply:
Removing seems fine to me, no strong feelings.